ValveResourceFormat/Resource/ResourceTypes/Texture.cs

```diff
@@ -161,11 +163,11 @@ public Bitmap GenerateBitmap()
                             }
                         }
 
-                        return ThirdParty.DDSImage.UncompressDXT5(Reader, Width, Height);
+                        return ThirdParty.DDSImage.UncompressDXT5(Reader, Width, Height, ycgco, Flags.HasFlag(VTexFlags.SUGGEST_CLAMPU));
```

Comment:
I know, it's prepared for the discussed function in GenerateBitmap that would read the REDI data, but just now it's just set to false.